doc/classes/OpenXRAPIExtension.xml

```diff
@@ -0,0 +1,109 @@
+	</brief_description>
+	<description>
+		[OpenXRAPIExtension] makes OpenXR available for GDExtension. It provides the OpenXR API to GDExtension through the [method get_instance_proc_addr] method, and the OpenXR instance through [method get_instance].
+		It also provides methods for querying the status of OpenXR initialization, and helper methods for ease of use of the API with GDExtension.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="can_render">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if OpenXR is initialized for rendering with an XR viewport.
+			</description>
+		</method>
+		<method name="get_error_string">
+			<return type="String" />
+			<param index="0" name="result" type="int" />
+			<description>
+				Returns an error string for a given [code]XrResult[/code].
+			</description>
+		</method>
+		<method name="get_instance">
+			<return type="int" />
+			<description>
+				Returns the [code]XrInstance[/code] craated at the initialization of [code]OpenXRAPI[/code].
+			</description>
+		</method>
+		<method name="get_instance_proc_addr">
+			<return type="int" />
+			<param index="0" name="name" type="String" />
+			<description>
+				Returns the function pointer of the OpenXR function with the specified name, cast to an integer.
+				[code]openxr/util.h[/code] contains macros for ease of use in acquiring OpenXR functions, e.g. [code]GDEXTENSION_INIT_XR_FUNC_V(xrCreateAction)[/code].
+				If the function with the given name does not exist, the method returns [code]0[/code].
+			</description>
+		</method>
+		<method name="get_next_frame_time">
+			<return type="int" />
+			<description>
+				Returns the timing for the next frame.
+			</description>
+		</method>
+		<method name="get_play_space">
+			<return type="int" />
+			<description>
+				Returns the play space, which is an [code]XrSpace[/code] cast to an integer.
+			</description>
+		</method>
+		<method name="get_session">
+			<return type="int" />
+			<description>
+				Returns the OpenXR session, which is an [code]XrSession[/code] cast to an integer.
+			</description>
+		</method>
+		<method name="get_swapchain_format_name">
+			<return type="String" />
+			<param index="0" name="swapchain_format" type="int" />
+			<description>
+				Returns the name of the specified swapchain format.
+			</description>
+		</method>
+		<method name="get_system_id">
+			<return type="int" />
+			<description>
+				Returns the id of the system, which is a [code]XrSystemId[/code] cast to an integer.
+			</description>
+		</method>
+		<method name="is_initialized">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if OpenXR is initialized.
+			</description>
+		</method>
+		<method name="is_running">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if OpenXR is running ([code]xrBeginSession[/code] was successfully called and the swapchains were created).
```

Comment:
```suggestion
				Returns [code]true[/code] if OpenXR is running (i.e. [code]XrBeginSession[/code] was successfully called and the swapchains were created).
```

I assume it needs to be capitalized like all other `Xr` codes?

Reply:
@YuriSizov  [xrBeginSession](https://registry.khronos.org/OpenXR/specs/1.0/man/html/xrBeginSession.html) is an OpenXR function, so by the used naming convention it shouldn't be capitalized.